client/src/assets/scss/main.scss

```diff
@@ -8,7 +8,93 @@
   box-sizing: border-box;
 }
 
+body {
+  background-color: #EDF6F9;
+}
+
 :root {
   -moz-tab-size: 4;
   tab-size: 4;
 }
+
+.page-header {
+  text-align: left;
+  display: inline;
+  color: #006D77;
+}
+
+.small-4 {
```

Comment:
this is ok, but I would caution about a lot of changes made to existing styles from Foundation - `small-4` is a built in class that will likely be used in many places in the app to help with alignment, so pairing it with additional styling that dictates a background color might be too much in one place. What I would suggest is creating a new class, `box-element` or what ever you want to call it, and have that apply your other styling like colors, separate from the alignment that `small-4` is intended to specify